apps/native-component-list/src/screens/Contacts@Next/ContactsNextFormScreen.tsx

```diff
@@ -0,0 +1,145 @@
+import React, { useEffect, useState } from 'react';
+import { View, StyleSheet, Text, ScrollView } from 'react-native';
+import Button from '../../components/Button';
+import { Contact } from 'expo-contacts/next';
+
+export default function ContactApiScreen() {
+  const [contact, setContact] = useState<Contact | null>(null);
+  const [status, setStatus] = useState<string>('');
+  const [loading, setLoading] = useState(false);
```

Comment:
```suggestion
  const [loading, setLoading] = useState(false);
```

not a huge deal, given this is a tiny example screen, but considering there's a `status` state, it feels natural it'd also handle the `loading` state at the same time